plugins/ScheduledReports/angularjs/manage-scheduled-report/manage-scheduled-report.controller.js

```diff
@@ -177,6 +177,37 @@
             resetParameters(this.report.type, this.report);
         };
 
+        this.displayReport = function (url) {
+            // Split the URL into its parts
+            var urlParts = url.split('?');
+            var basePath = urlParts[0];
+            var params = urlParts[1].split('&');
+
+            var formContent = '';
+            var separator = '?';
+            $.each(params, function(key, queryStr) {
+                var queryParts = queryStr.split('=');
+                if (queryParts[0] == 'token_auth') {
+                    // Hide the token_auth in an input element
+                    formContent = $('<input>')
+                        .attr('type', 'hidden')
+                        .attr('name', queryParts[0])
+                        .attr('value', queryParts[1]);
+                } else {
+                    // Everything else can be exposed to the user in the URL
+                    basePath += separator + queryStr;
+                    separator = '&';
+                }
+            });
+
+            // Build and submit the form
+            // The target="_blank" only works if the form's id attribute is also set
+            $('<form action="' + basePath + '" method="POST" target="_blank" id="downloadReportForm"></form>')
```

Comment:
be good to set basePath also through `attr()` otherwise we may run into XSS again (not sure that this would fully prevent it though)